demos/supabase/README.md

```diff
@@ -0,0 +1,29 @@
+# Supabase Local Development
+
+This is demonstration of running Supabase locally in tandem with our [React Supabase Todolist demo](https://github.com/powersync-ja/powersync-js/tree/main/demos/react-supabase-todolist).
+
+## Getting Started
+
+Follow the [instructions](https://supabase.com/docs/guides/cli/getting-started) for configuring Supabase locally.
+
+Copy the environment variables template file
+
+```bash
+cp .env.template .env
+```
+
+Start the Supabase project
+
+```bash
+supabase start
+```
+
+Once started the console will contain details for the Supabase Anon key and Postgres connection. Apply these values to the `.env` file.
```

Comment:
This was a tad confusing as I didn't need to apply any changes related to Postgres, just the Anon key.

Reply:
This has now been updated to be more specific